.prettierignore

```diff
@@ -8,3 +8,4 @@
 build/**
 webpack-stats.json
 .mypy_cache/**
+.ruff_cache/**
```

Comment:
Does Prettier ever complain about this cache?

Reply:
No, it just takes a couple of seconds on each run.